conformance/utils/suite/suite.go

```diff
@@ -166,8 +168,16 @@ const (
 
 // NewConformanceTestSuite is a helper to use for creating a new ConformanceTestSuite.
 func NewConformanceTestSuite(options ConformanceOptions) (*ConformanceTestSuite, error) {
-	if len(options.ConformanceProfiles) == 0 {
-		return nil, errors.New("conformance profiles need to be given")
+	// test suite callers are required to provide either:
+	// - one conformance profile via the flag '-conformance-profiles'
+	// - a list of supported features via the flag '-supported-features'
+	// - an explicit list of tests to run via the flag '-run-tests'
```

Comment:
It's possible to specify a single test through the run-test flag. I think it makes sense to move the `--run-test` flag to `--run-tests` and give the users the possibility to specify a comma-separated list of tests though

Reply:
fixed the comment - the functionality you described would be a good first issue

Reply:
yep yep, I'll open an issue for that 👍